lib/jekyll-github-metadata/client.rb

```diff
@@ -59,9 +59,9 @@ def method_missing(method_name, *args, &block)
 
       def save_from_errors(default = false, &block)
         if block.arity == 1
-          block.call(@client)
+          yield(@client)
         else
-          block.call
+          yield
```

Comment:
wont this be a problem? What happens to `block` here? Is yield supposed to know to call block? :confused: